docs/load-balancer-annotations.md

```diff
@@ -78,20 +80,22 @@ Note:
+| Name                                                                       | Description                                                                                                                                                                                  | Default                                   |
+|----------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
+| `oci-network-load-balancer.oraclecloud.com/internal`                       | Create an [internal network load balancer][1]. Cannot be modified after load balancer creation.                                                                                              | `false`                                   |
+| `oci-network-load-balancer.oraclecloud.com/subnet`                         | The OCID of the required regional or AD specific subnet to attach the network load balancer.	                                                                                                | Value set for the cluster                 |
+| `oci-network-load-balancer.oraclecloud.com/oci-network-security-groups`    | Specifies Network Security Groups' OCIDs to be associated with the network load balancer.	                                                                                                   | `""`                                      |
+| `oci-network-load-balancer.oraclecloud.com/initial-freeform-tags-override` | Specifies one or multiple Freeform tags to apply to the OCI Network Load Balancer. Valid values: `'{"tag1": "value1", "tag2": "value2"}'`                                                    | `""`                                      |
+| `oci-network-load-balancer.oraclecloud.com/initial-defined-tags-override`  | Specifies one or multiple Defined tags to apply to the OCI Network Load Balancer.  Valid values: `'{"namespace1": {"tag1": "value1", "tag2": "value2"}}'`                                    | `""`                                      |
+| `oci-network-load-balancer.oraclecloud.com/health-check-retries`           | The number of retries to attempt before a backend server is considered "unhealthy".	                                                                                                         | `3`                                       |
+| `oci-network-load-balancer.oraclecloud.com/health-check-timeout`           | The maximum time, in milliseconds, to wait for a reply to a health check. A health check is successful only if a reply returns within this timeout period.                                   | `3000 ms`                                 |
```

Comment:
Value should be `3000`. In description mention the timeout should be given in millisecond

Reply:
Updates made.